src/SFML/Window/Win32/ClipboardImpl.cpp

```diff
@@ -35,6 +35,26 @@
 
 #include <cstring>
 
+namespace
+{
+std::string getErrorString(DWORD error)
+{
+    PTCHAR buffer = nullptr;
+
+    if (FormatMessage(FORMAT_MESSAGE_MAX_WIDTH_MASK | FORMAT_MESSAGE_ALLOCATE_BUFFER | FORMAT_MESSAGE_FROM_SYSTEM,
+                        nullptr,
+                        error,
+                        0,
+                        reinterpret_cast<PTCHAR>(&buffer),
+                        0,
+                        nullptr) == 0)
+        return "Unknown error.";
+
+    sf::String message = buffer;
+    LocalFree(buffer);
+    return message.toAnsiString();
```

Comment:
We now have 3 nearly identical implementations of this function in src/Window/Win32/ClipboardImpl.cpp, src/Window/Win32/JoystickImpl.cpp, and src/Window/Win32/WglContext.cpp. I propose we add a new src/Window/Win32/Utils.hpp file where we can this function. This will reduce code duplication and avoid these unity builds failures we're getting. There is already precedent for this. We already have a src/Window/Unix/Utils.hpp header for some X11 utilities.